.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,14 @@
+# Proposed Changes
+
+<!--
 Hi there. Thanks for submitting a pull request!
 
 Please describe your changes in this PR description.
 You can also include a link to an active Jira ticket that provides relevant context if there is one.
 
-https://policies.zephyrai.bio/sdlc.html
+https://zephyrai.atlassian.net/wiki/spaces/HELIX/pages/37650458/Secure+Software+Development+and+Product+Security
```

Comment:
NIT: This should probably link to a more basic page, which then links to the security page.

Reply:
This is our SDLC process documentation. I replaced the dead link to `policies.zephyrai.bio` with a live link to our internal wiki.